core/storage/email/gae_models.py

```diff
@@ -365,8 +365,9 @@ def get_by_reply_to_id(cls, reply_to_id):
             reply_to_id: str. The unique 'reply-to' id.
 
         Returns:
-            str or None. The FeedbackEmailReplyToIdModel instance corresponding
-                to the given 'reply-to' id if it is fetched else None.
+            The FeedbackEmailReplyToIdModel instance corresponding to
+            the given 'reply-to' id it is present in the datastore.
```

Comment:
```suggestion
            the given 'reply-to' id if it is present in the datastore.
```

Reply:
There, this should be better. Indeed, I added the "if" that I forgot